moduleframework/tools/modulelint.py

```diff
@@ -71,25 +71,96 @@ def test_run_or_usage_label_exists(self):
         self.assertTrue(label_found)
 
 
+class DockerfileLinterInContainer(container_avocado_test.ContainerAvocadoTest):
+    """
+    :avocado: enable
+
+    """
+
+    def _file_to_check(self, doc_file_list):
+        test_failed = False
+        for doc in doc_file_list.split('\n'):
+            exit_status = self.run("test -e %s" % doc, ignore_status=True).exit_status
+            if int(exit_status) == 0:
+                self.log.debug("%s doc file exists in container" % doc)
+                test_failed = True
+        return test_failed
+
+    def test_all_nodocs(self):
+        """
+        This test checks all docs installed in container. As delivered by Platform
+        as delivered by installing packages RUN command
+        :return: WARNING in case we found some docs
+                 PASS in case there is no doc file found
+        """
+        self.start()
+        all_docs = self.run("rpm -qad", verbose=False).stdout
+        test_failed = self._file_to_check(all_docs)
+        if test_failed:
+            self.log.warn("Documentation files exist in container. They are installed by Platform or by RUN commands.")
+        self.assertTrue(True)
+
+    def test_installed_docs(self):
+        """
+        This test checks whether no docs are installed by RUN dnf command
+        :return: FAILED in case we found some docs
+                 PASS in case there is no doc file found
+        """
+        self.start()
+        installed_pkgs = self.run("rpm -qa --qf '%{{NAME}}\n'", verbose=False).stdout
```

Comment:
this is wrong, you should not check %{name}. Check the full NEVRA because this is only one unique things

Reply:
Yeah, it is a stupid typo. I will fix it with %{NAME} based on rpm man page.

Reply:
Unfortunately test takes a pretty much time. Around 150s which is not good. Based on Travis tests.

Reply:
See reason of double brackets:
https://github.com/fedora-modularity/meta-test-family/blob/devel/moduleframework/common.py#L133
https://github.com/fedora-modularity/meta-test-family/blob/devel/moduleframework/common.py#L384